src/Api/ItemList.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace OpsWay\ZohoBooks\Api;
+
+
+class ItemList implements \ArrayAccess, \IteratorAggregate
+{
+    /**
+     * @var array
+     */
+    private $items = [];
+
+    /**
+     * @var array
+     */
+    private $pageContext = [];
+
+    /**
+     * @param array $items
+     * @param array $pageContext
+     */
+    public function __construct(array $items, array $pageContext)
```

Comment:
There is no reason to make it optional because zoho do not allow lists without pagination.